package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@articulate/authentic",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Bumping the version isn't done until after the updates have been merged into `#master`.  After the PR is merged, a new version is created and published with the following flow:

```sh
> yarn version  # bumps the version and creates a matching git tag
> git push --tags origin master
> npm publish --access=public
```
Just move this back to `0.1.2` for now.